VideoWeb/VideoWeb/ClientApp/src/app/shared/footer/footer.component.scss

```diff
@@ -16,23 +16,23 @@
   white-space: nowrap;
 }
 
-.govuk-footer__copyright-logo {
+.govuk-footer-background-uk {
   background-image: url('/assets/images/govuk-crest.png');
 }
 
-.govuk-footer__copyright-logo-scottish {
-  background-image: url('/assets/images/scottish-crest-footer.png');
+.govuk-footer-background-scottish {
+  background-image: url('/assets/images/scottish-crest-grey.png');
 }
 
 @media only screen and (-webkit-min-device-pixel-ratio: 2),
   only screen and (min-resolution: 192dpi),
   only screen and (min-resolution: 2dppx) {
-  .govuk-footer__copyright-logo {
+  .govuk-footer-background-uk {
     background-image: url('/assets/images/govuk-crest-2x.png');
   }
```

Comment:
You should be able to nest this media query above line 21 just to structure it better. And remove the class here.

Reply:
could you double check to make sure thats what you meant, I didnt know actually you could next media queries inside of the class black

Reply:
It's a feature of SCSS. I believe that wouldn't work with regular CSS